source/libCoap/src/sn_coap_builder.c

```diff
@@ -1158,3 +1195,41 @@ static void sn_coap_builder_payload_build(uint8_t **dst_packet_data_pptr, sn_coa
         (*dst_packet_data_pptr) += src_coap_msg_ptr->payload_len;
     }
 }
+
+static uint16_t sn_coap_builder_block_option_size(sn_coap_hdr_s *src_coap_msg_ptr,
+                                                  sn_coap_option_numbers_e option,
+                                                  bool is_blockwise_needed)
+{
+    (void) src_coap_msg_ptr;
+    (void) option;
+    (void) is_blockwise_needed;
```

Comment:
These three lines might be moved to #else -block, as now they may suppress the "unused variable" -warning from the "#if YOTTA_CFG_COAP_MAX_BLOCKWISE_PAYLOAD_SIZE" -branch also.